core/src/main/scala/kafka/tools/TestRaftServer.scala

```diff
@@ -52,6 +52,7 @@ class TestRaftServer(
   import kafka.tools.TestRaftServer._
 
   private val partition = new TopicPartition("__cluster_metadata", 0)
+  private val topicId = Uuid.METADATA_TOPIC_ID
```

Comment:
I'm a tad concerned about reusing the same topic id in a different context. Maybe instead we could hard-code a different sentinel and use it only here, say `new Uuid(0L, 2L)`. Then we can also change the topic name to `__raft_performance_test` or something like that. That should be enough to prevent any potential conflicts.

Reply:
I was wondering about this too. In another test, (for MockLog) I used a random Uuid. Is there something we get from the hardcoded one vs random?

Reply:
Yes, we need a consistent value so that we can run the test raft server as a raft cluster.